design/k8s-job-hook.md

```diff
@@ -0,0 +1,233 @@
+# Kubernetes Job Hook Proposal
+
+## Goals
+
+- Enable hooks that trigger execution of some behavior directly by the lakeFS server using Kubernetes
```

Comment:
Initially I read this as allowing K8s to trigger execution of some lakeFS behaviour.  I think this phrasing is indeed ambiguous, perhaps this will be better?
```suggestion
- Allow lakeFS actions directly to trigger actions running on Kubernetes
```